bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/smarthome/Constants.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.amazonechocontrol.internal.smarthome;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.thing.type.ChannelTypeUID;
+import org.openhab.binding.amazonechocontrol.internal.AmazonEchoControlBindingConstants;
+
+/**
+ * @author Michael Geramb - Initial contribution
+ */
+@NonNullByDefault
+public class Constants {
+    public static final Map<String, @Nullable Function<String, HandlerBase>> HANDLER_FACTORY = new HashMap<>();
```

Comment:
What is the point of this using `Function<String, HandlerBase>`  values instead of `Supplier<HandlerBase>` values? None of the statically constructed values even use the String argument.

Reply:
Done